models/like-dislike.js

```diff
@@ -0,0 +1,50 @@
+const db = require('../database/db');
+
+const likeDislike = {
```

Comment:
Hi Sid, could you please consider changing this object/class name to 'Votes' (or another single name that helps describe what this object/class does)  as `likeDislike.likeDislike` doesn't read very well

Reply:
Done! Thanks Chris!